lighthouse-core/scripts/build-report-for-autodeployment.js

```diff
@@ -14,6 +14,13 @@ const path = require('path');
 const ReportGenerator = require('../../lighthouse-core/report/report-generator.js');
 const lhr = /** @type {LH.Result} */ (require('../../lighthouse-core/test/results/sample_v2.json'));
 
+lhr.categories.test = {
```

Comment:
key on `categories` should be the same as `id`. Not sure we actually depend on that anywhere in the report renderer, but just in case...

can you also add a comment (`// Add a plugin to demo plugin rendering` or whatever) and drop a note in #8159 for the feature request?

Reply:
done